README.md

```diff
@@ -79,3 +68,20 @@ terraform init --upgrade
 ```console
 terraform apply
 ```
+
```

Comment:
"Test the provider"? Why it is test? It should be a tutorial, not test.

What happened after test? What user should expect? How to check the job is running? How to get results?

Reply:
good point; fixed.